pkg/libvirttools/virtualization.c

```diff
@@ -20,16 +20,64 @@ limitations under the License.
 #include "virtualization.h"
```

Comment:
Yes, for me there are two major reasons for C wrappers:

* I couldn't find any good way to convert to `const char*` in Go code by cgo
* in case of images, I had to write a function https://github.com/Mirantis/virtlet/blob/master/pkg/libvirttools/image.c#L27-L37 which has to be used as an argument to the actual libvirt function https://github.com/Mirantis/virtlet/blob/master/pkg/libvirttools/image.c#L56 and IMO writing that function in a comment in Go source file would be awful

Another reason (but not so important) is that I try to avoid writing huge functions in Go code which would contain a lot of `C.something` statements. I just think that moving them into C wrappers makes it more readable. The only exception I had to do with this approach in this PR is `ListPodSandbox` function, where I'm converting `*virDomainPtr` to a slice of `*kubeapi.Container` - I coudn't find any good looking way of wrapping that by C.

I know that images code isn't very consistent with what I wrote in the paragraph above, but I'm planning to fix it. But here I did better, hopefully.